content/_plugins.md

```diff
@@ -7,19 +7,24 @@
 Attribute | Type | Description
 --------- | ------- | -------
 `archive_name` | string | The plugin archive file name.
+`blueprint_labels` | dict | The blueprint labels which describe blueprints using this plugin.
+`created_by` | string | The user who created the plugin.
 `distribution_release` | string | The OS distribution release name the plugin was compiled on. 'None' in-case the plugin is cross platform.
 `distribution_version` | string | The OS distribution version the plugin was compiled on. 'None' in-case the plugin is cross platform.
 `distribution` | string | The OS distribution the plugin was compiled on. 'None' in-case the plugin is cross platform.
 `excluded_wheels` | list | a list of wheels that were excluded from the plugin package.
 `id` | string | The ID assigned to the plugin upon upload.
+`installation_state` | list | The state of plugin installation (usually either empty list or '["installed"]').
```

Comment:
Should we be documenting this and created_by? Are they not internal only?

Reply:
I wasn't sure, maybe indeed they are.

Reply:
(however `curl`ing the endpoint will get you these)

Reply:
Fair enough, let's go with this then.